bin/wmagent-resource-control

```diff
@@ -167,13 +167,12 @@ def printThresholds(resourceControl, desiredSiteName):
         print "%s - %d running, %d pending, %d running slots total, %d pending slots total%s:" % (
                         siteName, runningJobs, pendingJobs, runningSlots, pendingSlots, stateMsg)
 
-        for task in siteThresholds['thresholds']:
+        for task, thres in siteThresholds['thresholds'].iteritems():
```

Comment:
I get your point :-). My point is that using items() is the more "pythonic" way to write this (hence the change in python3). This has nothing to do with functionality, this has to do with what looks natural in code. And "for a,b in items()" look natural, so natural that the majority of our existing code is using it for iterations. Blindly converting it to "for a,b in [items()]" is wrong.

Before we run the converter we should have a campaign to switch all items() call that we can (as I said, I expect 100%) to iteritems() and only then run the converter.